### ISSUE TITLE
Segementation fault when creating a new project

### ISSUE THREAD

**Issue by idawettergren** [sgoldenlab/simba#356]
When I create a new project and try to browse for the project directory or add predictive classifiers simba crashes with a "zsh: segmentation fault simba". I can write in the text boxes and change the drop-down menus under animal settings without issue.

OS: Sonoma 14.4.1 macOS.
Python version: 3.10
Not using anaconda

**Comment by idawettergren** [comment]
Never mind, updated to simba 1.90.3 and now it works

**Comment by sronilsson** [comment]
Thanks for reporting @idawettergren !

**Comment by idawettergren** [comment]
I unfortunately still have the same issue. After updating simba I could create a new project, upload videos and an .slp file but when I pressed c to assign the tracks the program crashed with another zsh: segmentation fault and then kept crashing with the same error when I tried to browse for the project configuration file

**Comment by sronilsson** [comment]
Thanks @idawettergren - I don’t have access to a machine running Sonoma 14.4.1 Mac OS - closest is Ventura 13.4. I tested importing multi-animal sleap H5 on there using python 3.10 and 3.6, and I did hit a related error in python3.10, not with segmentation but. “[Python app crashed with "PyEval_RestoreThread: the function must be called with the GIL held, but the GIL is released](https://stackoverflow.com/questions/66795159/python-app-crashed-with-pyeval-restorethread-the-function-must-be-called-with)” that I should look into, but it ran in python 3.6. 

If you sing python 3.6 - how does it run on your end?

**Comment by idawettergren** [comment]
If I install and run simba in a venv environment with python 3.6.15 I still get the same error. I also for some reason still get the warning on the homepage that I'm using python 3.10

**Comment by sronilsson** [comment]
Hi @idawettergren ! Yes the warning message makes me think it is still possible it isn't running 3.6. 

If you activate the venv and run `python --version`, as below, do you see python 3.6? 

<img width="403" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/4525776c-5b5f-484d-a145-ecaf5f8c21aa">

**Comment by idawettergren** [comment]
Yes I see "Python 3.6.15". Without changing anything it is now working again, even if I still see the warning message. Will update here when I manage to start the training (or if it crashes again before then)

**Comment by sronilsson** [comment]
If it crashes, would you mind trying it out in conda and I can troubleshoot with you? I don't work much in venv, and it might be that you have to troubleshoot venv with me at the same time as I troubleshoot segment fault with you :)
